static/sass/_pattern_global-nav.scss

```diff
@@ -0,0 +1,105 @@
+@mixin canonical-p-global-nav {
+
+  // Using id to override inherited styles from Vanilla and project
+  #g-navigation {
+    height: 2rem;
+    line-height: 1rem;
+    background-color: $color-dark;
```

Comment:
For some reason this line isn't resulting in `background-color: #111`

Reply:
ok I found the issue... the rebranding of canonical.com changed this variable https://github.com/canonical/canonical.com/blob/c514c8788a8f80b326f9779222359d7bd811aaf7/static/sass/styles.scss#L7